sdks/python/apache_beam/runners/portability/local_job_service.py

```diff
@@ -235,6 +207,29 @@ def cancel(self):
       # TODO(robertwb): Actually cancel...
       self.state = beam_job_api_pb2.JobState.CANCELLED
 
+  def get_state_stream(self):
+    # Register for any new state changes.
```

Comment:
It'd be good, here and in the message stream, to start off with a message indicating the current state.

Reply:
Done